content-service.jenkinsfile

```diff
@@ -0,0 +1,94 @@
+pipeline {
+    agent any
+    environment
+    {
+      // NOTE: this file must be executed in a directory whose name is a numeric sequence, and whose parent is named "Releases".
+      // This is how other Jenkinsfiles in the Release process determine the current release number.
```

Comment:
Is there a README file for this PR where this comment can be included?  Since it occurs in all 3 Jenkins files, maybe it's best to move it up to general documentation?

Reply:
Jenkinsfiles are imported directly into Jenkins. If something goes wrong, it's most likely that the user will go straight to the Jenkinsfile and see this note there. Moving this to a README makes it less visible for the possible situation(s) where it's needed, IMO. It could be _added_ to a README, but I don't think it should be removed from here.

Reply:
Okay - I leave it up to you if you think it should be added to a README in addition to being in the Jenkinsfile.